sound.js

```diff
@@ -31,7 +39,7 @@ function Sound(filename, basePath, onError) {
   this._pan = 0;
   this._numberOfLoops = 0;
   this._speed = 1;
-  RNSound.prepare(this._filename, this._key, (error, props) => {
+  RNSound.prepare(this._filename, this._key, enableSMTCIntegration, (error, props) => {
```

Comment:
What happens on iOS and Android here? Will the RNSound.prepare method receive the callback as argument 4 rather than 3 as expected?  I haven't tried it but this may cause problems - I suspect the signature of the iOS and Android prepare methods would need adjustment.

It may be better to pass options as a JS object here directly so we can add other options later in native code without needing to rework this method.